lib/routes/home/widgets/payments_list/dialog/payment_details_dialog_amount.dart

```diff
@@ -7,12 +7,12 @@ import 'package:c_breez/models/currency.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-class PaymentDetailsAmount extends StatelessWidget {
```

Comment:
Just a minor fix on the name to fit breezmobile component and its file name

Reply:
I suggest we use shorter and singular names for items we display on PaymentDetailsDialog in the form of: PaymentDetail{Parameter}.
_PaymentDetailTitle, PaymentDetailPreimage, PaymentDetailAmount etc._

Reply:
I want to keep the same name on BreezMobile. There are more widgets, and the more verbose name has a reason to differentiate from other widgets

Reply:
Last time I checked these were not extracted as standalone widgets on Breezmobile but I see what you mean, it's just a bit tiring on the eyes that's all :) They can stay as they are.

Reply:
You are right; the PR is still open https://github.com/breez/breezmobile/pull/1048

If you want to suggest names there, I can apply the same names here later; the only thing is there are more widgets so the naming can be thought with better context.